README.md

```diff
@@ -19,11 +35,65 @@ def deps do
 end
 ```
 
-## Running tests
-Clone the repository and fetch its dependencies:
+## Getting started
+
+You will need to create an API key via your [incident.io
+dashboard](https://app.incident.io/settings/api-keys) to make requests. Please
+note their warning message about the scope for this API key:
+
+> When you create the key, you'll be able to choose what actions it can take for
+> your account: choose carefully, as those roles can only be set when you first
+> create the key.
+
+Once you've created an API key then you're ready to start making requests.
+
+## Example
+
+Assuming that you've created and configured your incident.io API key as an
+environment variable named `INCIDENT_API_KEY`, you can
+create and read incidents as simply as the following example module:
+
+```elixir
+defmodule MyIncidentIo do
+  use IncidentIo
+
+  alias IncidentIo.Client
+  alias IncidentIo.IncidentsV2
+
+  def create(idempotency_key)
+    IncidentsV2.create(client, idempotency_key: idempotency_key, visibility: :public)
+  end
+
+  def read(incident_id)
+    IncidentsV2.show(client, incident_id)
+  end
+
+  defp client
+    Client.new(%{api_key: System.fetch_env!("INCIDENT_API_KEY")
+  end
+end
+```
```

Comment:
Is this useful? May need to elicit some external feedback or wait for it before attempting to rework it.